NGCHM/src/mda/ngchm/util/NGCHM_AppGenerator.java

```diff
@@ -127,6 +127,9 @@ public static void main(String[] args) {
     				bw.write("<style type='text/css'>");
      				bw.write(styleToString(args[0], cssFile));
      				bw.write("</style>\n");
+			} else if (line.contains("ngChmIcon.ico")) {
+				final String ngChmIconBase64 = "iVBORw0KGgoAAAANSUhEUgAAABIAAAASCAYAAABWzo5XAAAAIGNIUk0AAHomAACAhAAA+gAAAIDoAAB1MAAA6mAAADqYAAAXcJy6UTwAAAAGYktHRAD/AP8A/6C9p5MAAAO4SURBVDjLbZTbT5t1GMc/v/ZtaWmhhUE5FlhB5BA3p5KYOC90icTDhpkxmZioxItFL7wxMWqMyW4WXXZhptGZZUu2yIwiRiKwRYYYmFsEItvAwBwDhK20nDpeSo9v38eLzjs/f8A3eb6HR4FIXh58+hkElsdpjX8Lv4zA2Bh9Xy0S8vp584kZMoYTvaQaZziIY2YCdj8M5eVUVsDdoEKr8gseDyz+oygWO5T6wGYDwKUl8eQAhokyTZSCDBoJixtbWrASp8rvIG0oEIlJQo8LiLx4ULK0t4uASE+XyOSoyI0bIjdnRERkeUVk4HeR0NXbIiOXJLJhyEZCRDt7WiO2Lbz0jM6zDWvw5x2oqICWFqjYCRUBOH6MqMfP6aIHKcoBhxuSbgNyNYzBYQwDFIj4vCnCkZswOANnfoL9+6G+Hh7ZBWigFKtaKT5jmT174NU34KB/gp2BMHzyOaytowFklEbXN/WUXZlk78Ue2LcP6irp6xHSLujnJKm8MoiC1QqxLTj2fQ2R5VJeaPwAty+VFdKjFgYnc3h8Ms7ejW2wJiHfx5VZ2IzDKethEMACQRPCJpwbLyA2C/JQGb4i0NraIJmE7u8g31EMTV46+8u5MA1Hfm6jJD7HuzUlWNNJzHvz2G7l4Trv5f1IjASbFFo7sJje+6dlYO0OxIqSsGOThJ4iuAr+4B/YN8O4vbNgGIABW8AMeLlPZgakGDUfFNE34ew5KPJDZTN0fgm/9cNbH4HmgA/fgeRiktKAg8R7R8gc/ZgvuuDiAOwOgN0KWuL6AqkouO+CJwM+KwR0WCVFudRiw0o6AmZoFZwOLFEdUlC5FGbXSoLaEieaTaEka+P/EhqaJlHfwMIcuPVVHvv6tWy36gLQ3Qlj1+DJRyHXiUZ7O6apWI/aySWGSyIE9QLWQyly7DuwmxAKQoHFAa2tmA4PFrcLnnoeAo0sl7VgaA6UiAjA2CLUqDjF5hgjf7cwvODk5aehpMjkxAkLlVXQ8TpgbMHcFOTkg6YYuN2EngTt+NEUNrvCX2djeAqu/VpL1A5rMajLF6qrFb29UF0NjQ0wcdXGpe5anjtgpblJSBgmGha0Mz/YyS+AVzpg6C8nfUMV5LpAs8KteUVGwegoLC3BoUNwedzBj5cdeBvAWweefLBpoJbPD4rSdXw9pzgZPMDb1w9jsYBpwlTvPI1VMabtzQQXobUVCgtBqWwYItmPoxRopQXbgA735nElN4CsCIDbnsZiS1D3QLaLIhCNgs8HKyvZRfzHvwf8m2NjhD8kAAAAJXRFWHRkYXRlOmNyZWF0ZQAyMDIzLTAzLTEwVDE3OjExOjQ0KzAwOjAwryH/YQAAACV0RVh0ZGF0ZTptb2RpZnkAMjAyMy0wMy0xMFQxNzoxMTo0NCswMDowMN58R90AAAAodEVYdGRhdGU6dGltZXN0YW1wADIwMjMtMDMtMTBUMTc6MTE6NDUrMDA6MDAvHm22AAAAAElFTkSuQmCC";
```

Comment:
Why isn't this handled by the next else-case, that reads and base64 encodes an image file?  Do you really want this image hard-coded?

Reply:
We'd have to move and convert ngChmIcon.ico to images/ngChmIcon.png. I imagine that would work.